packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/Actions.tsx

```diff
@@ -292,195 +301,193 @@ const BulkActionModal = ({
+            {formatMessage({
+              id: 'popUpWarning.bodyMessage.contentType.delete.all',
+              defaultMessage: 'What do you want to delete?',
+            })}
+          </Typography>
+          <Box textAlign="center" padding={3}>
+            <Flex onChange={handleChange} direction="column" alignItems="flex-start" gap={3}>
+              <Radio
+                checked={!isDeleteAllLocale}
+                value={BULK_DELETE_OPTIONS.SELECTED_LOCALE}
+                name="bulk-delete-options"
+              >
+                {formatMessage(
+                  {
+                    id: 'popUpWarning.bodyMessage.contentType.delete.selectedLocale',
+                    defaultMessage: 'Delete entries ({locale})',
+                  },
+                  { locale: currentLocale } //TODO: get locale name from locale code
+                )}
+              </Radio>
+              <Radio
+                checked={isDeleteAllLocale}
+                value={BULK_DELETE_OPTIONS.ALL_LOCALES}
+                name="bulk-delete-options"
+              >
+                {formatMessage({
+                  id: 'popUpWarning.bodyMessage.contentType.delete.allLocales',
+                  defaultMessage: 'Delete entries (all locales)',
+                })}
+              </Radio>
+            </Flex>
+          </Box>
+        </Flex>
+      ),
+      onConfirm: handleConfirmBulkDelete,
+    },
+  };
+};
+
+const UnpublishAction: BulkActionComponent = ({ documentIds, model }) => {
+  const { formatMessage } = useIntl();
+  const { schema } = useDoc();
+  const { selectRow, selectedRows } = useTable('unpublishAction', (state) => state);
+  const hasPublishPermission = useDocumentRBAC('unpublishAction', (state) => state.canPublish);
+  const hasI18nEnabled = Boolean(schema?.pluginOptions?.i18n);
+  const { unpublishMany: bulkUnpublishAction } = useDocumentActions();
+
+  const handleConfirmBulkUnpublish = async () => {
+    const data = await bulkUnpublishAction({ documentIds, model });
+    if (!('error' in data)) {
+      selectRow([]);
+    }
+  };
+
+  const showUnpublishButton =
+    hasPublishPermission && selectedRows.some((entry) => entry.status === 'published');
+
+  if (!showUnpublishButton) return null;
+
+  return {
+    actionType: 'unpublish',
+    variant: 'tertiary',
+    label: formatMessage({ id: 'app.utils.unpublish', defaultMessage: 'Unpublish' }),
+    dialog: {
+      type: 'dialog',
+      title: formatMessage({
+        id: 'app.components.ConfirmDialog.title',
+        defaultMessage: 'Confirmation',
+      }),
+      content: (
+        <Flex direction="column" alignItems="stretch" gap={2}>
+          <Typography id="confirm-description" textAlign="center">
+            {formatMessage({
+              id: 'popUpWarning.bodyMessage.contentType.unpublish.all',
+              defaultMessage: 'Are you sure you want to unpublish these entries?',
+            })}
+          </Typography>
+          {hasI18nEnabled && (
```

Comment:
I can't see this in the design, could it be removed?

Reply:
V4 design